lib/matplotlib/backends/backend_qt5.py

```diff
@@ -118,6 +118,11 @@ def _create_qApp():
                 if display is None or not re.search(r':\d', display):
                     raise RuntimeError('Invalid DISPLAY variable')
 
+            try:
+                QtWidgets.QApplication.setAttribute(
+                    QtCore.Qt.AA_EnableHighDpiScaling)
```

Comment:
This was inside a `if is_pyqt5()` block below; does running this still work fine on Qt4?

Reply:
that'll be caught by the AttributeError too.